VK/VK/GroupNewsScreen/GroupTableViewController.swift

```diff
@@ -147,5 +183,67 @@ final class GroupNewsTableViewController: UITableViewController {
     private func setupScene() {
         configureTableView()
         fetchPosts()
+        setupRefreshControll()
+    }
+
+    private func setupRefreshControll() {
+        refreshControl = UIRefreshControl()
+        refreshControl?.attributedTitle = NSAttributedString(string: Constant.refreshControllDefaultTitle)
+        refreshControl?.tintColor = .blue
+        refreshControl?.addTarget(self, action: #selector(refreshNews), for: .valueChanged)
+    }
+
+    @objc private func refreshNews() {
```

Comment:
```suggestion
    @objc private func refreshNews() {
```
objs методы вроде как в самом верху, нет?

Reply:
Сказали можно и внизу. Главное чтобы они были вместе а не в перемешку с обычными.